mobile/src/helpers/image/pick-image/pick-image.helper.ts

```diff
@@ -0,0 +1,16 @@
+import { Asset, launchImageLibrary } from 'react-native-image-picker';
+
+const pickImage = async (): Promise<Asset | undefined> => {
+  const res = await launchImageLibrary({
+    mediaType: 'photo',
+    selectionLimit: 1,
```

Comment:
Add constant for it and as default value into helper params

Reply:
Can you explain more, I don't quite understand what it should look like.

Reply:
I just need to return something like { uri, type, name } from the helper ?

Reply:
You should pass selectionLimit via props and in place where you use it you should have a constant for this